ui/app/templates/components/dashboard/vault-version-title.hbs

```diff
@@ -0,0 +1,46 @@
+<div class="has-top-margin-l has-bottom-margin-l is-flex-center">
+  <h1 class="title is-v-centered is-marginless" data-test-dashboard-version-header>
+    {{this.versionHeader}}
+  </h1>
+  <span class="has-left-margin-xxs">
+    <Hds::Button
+      @text="New"
+      @color="secondary"
+      @size="small"
+      @icon="wand"
+      class="hds-highlight-background hds-purple-text is-borderless is-shadowless is-underline has-text-weight-semibold new-features-modal-button"
```

Comment:
lol oops - was hoping turning into a button made _less_ styling changes necessary 😅

Reply:
lolol this might be removed when we get new designs back for the button :D